front/src/applications/operationalStudies/components/MacroEditor/consts.ts

```diff
@@ -159,3 +159,114 @@ export const DEFAULT_PACED_TRAIN_PAYLOAD: Pick<
 };
 
 export const DEFAULT_TIME_WINDOW = new Duration({ hours: 2 });
+
+const TRAIN_CATEGORY_TO_NGE: {
+  trainCategory: TrainCategory | 'NO_CATEGORY';
+  colorRef: string;
+}[] = [
+  { trainCategory: 'HIGH_SPEED_TRAIN', colorRef: 'HST' },
+  { trainCategory: 'INTERCITY_TRAIN', colorRef: 'IC' },
+  { trainCategory: 'REGIONAL_TRAIN', colorRef: 'RE' },
+  { trainCategory: 'COMMUTER_TRAIN', colorRef: 'S' },
+  { trainCategory: 'FREIGHT_TRAIN', colorRef: 'F' },
+  { trainCategory: 'FAST_FREIGHT_TRAIN', colorRef: 'FF' },
+  { trainCategory: 'TRAM_TRAIN', colorRef: 'TT' },
+  { trainCategory: 'WORK_TRAIN', colorRef: 'W' },
+  { trainCategory: 'NIGHT_TRAIN', colorRef: 'NT' },
+  { trainCategory: 'TOURISTIC_TRAIN', colorRef: 'TO' },
+  { trainCategory: 'NO_CATEGORY', colorRef: 'NC' },
+];
+
+export const OSRD_TRAINRUN_CATEGORY_MAPPING: Record<
+  TrainCategory | 'NO_CATEGORY',
+  TrainrunCategory
+> = TRAIN_CATEGORY_TO_NGE.reduce(
+  (acc, { trainCategory, colorRef }, index) => {
+    acc[trainCategory] = {
+      id: index + 1,
+      order: index + 1,
+      name: trainCategory,
+      shortName: '',
+      fachCategory: 'HaltezeitUncategorized',
+      colorRef,
+      minimalTurnaroundTime: 0,
+      nodeHeadwayStop: 0,
+      nodeHeadwayNonStop: 0,
+      sectionHeadway: 0,
+    };
+    return acc;
+  },
+  {} as Record<TrainCategory | 'NO_CATEGORY', TrainrunCategory>
+);
+
+export const NETZGRAFIK_COLOR_PALETTE = {
+  HST: 'red',
+  IC: 'purple',
+  RE: 'orange',
+  S: 'blue',
+  F: 'green',
+  FF: 'mint',
+  TT: 'black',
+  W: 'yellow',
+  NT: 'pink',
+  TO: 'brown',
+  NC: 'grey',
+} as const;
+
+export const CATEGORY_COLOR_VARIANTS = {
+  red: {
+    10: '#fae7e6',
+    50: '#e5221a',
```

Comment:
```suggestion
    50: '#e6221a',
```

https://www.sketch.com/s/86e75cbf-6bbe-4e87-a2d2-b63f33d1da1d/color-variables?search=red

Reply:
I have this in sketch :

<img width="911" alt="image" src="https://github.com/user-attachments/assets/e6354742-9af9-4956-b378-5d3308d72e7f" />

@thibautsailly , which one should i choose ?

Reply:
mmh that's weird indeed

![image](https://github.com/user-attachments/assets/6d96c168-4e29-49fe-b03c-faae5c989fb9)

Reply:
I'll keep `#e5221a` for now as it's the same value used in OSRD.
If @thibautsailly  prefers using #e6221a, we’ll update both values together later on.